### ISSUE TITLE
A11y review

### ISSUE THREAD

**Issue by mark-wiemer** [mark-wiemer/hello-hello#16]
Now that the site is shared broadly, it should be accessible.

**Comment by mark-wiemer** [comment]
Marking [SC 2.4.5 Multiple Ways](https://www.w3.org/WAI/WCAG21/Understanding/multiple-ways.html) as a pass, the links in the main content of the home page serve as another way of navigation.

**Comment by mark-wiemer** [comment]
A11y Insights 17.6 is the Text Spacing test, for reference

**Comment by mark-wiemer** [comment]
A11y Insights 23.5 [Target size](https://w3c.github.io/wcag/understanding/target-size-minimum.html) says 24x24 CSS pixels is good, can't overlap with others. [SC 2.5.5 Target Size (Enhanced) (Level AAA)](https://w3c.github.io/wcag/understanding/target-size-enhanced.html) says 44x44

**Comment by mark-wiemer** [comment]
Two bugs filed, hope to fix them today:

- #34 
- #44

Otherwise home page is looking great :)

**Comment by mark-wiemer** [comment]
About page looks good :)

**Comment by mark-wiemer** [comment]
Resume now looks good :)

**Comment by mark-wiemer** [comment]
Software page looks good. Closing!
